src/components/YouTubeShort.tsx

```
@@ -0,0 +1,19 @@
+interface Props {
+  id: string
+}
+
+export default function YouTubeShort({ id }: Props) {
+  return (
+    <div className="flex justify-center my-4">
```

ESLint

[error] Replace `flex·justify-center·my-4` with `my-4·flex·justify-center`

(prettier/prettier)